code-challenges.test.js

```diff
@@ -52,5 +115,38 @@ const hand2 = [5, 5, 3, 3, 4]
 const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe("fullHouse", () => {
+    it("takes in an array of 5 numbers and determines whether or not the array is a 'full house'", () => {
+        expect(fullHouse(hand1)).toEqual(true)
+        expect(fullHouse(hand2)).toEqual(false)
+        expect(fullHouse(hand3)).toEqual(false)
+    })
+})
 
 // b) Create the function that makes the test pass.
+
+// Pseudo Code:
+// Create a function that takes an array of five numbers as a parameter. (fullHouse)
+// Create a variable that invokes the sort method on the given parameter. (sortedHand)
+    // Within sort, use two elements as parameters that will return the second element subtracted from the first; and this arrow function will serve as a way to compare each value and sort appropriately.
+// Create if statement that will evaluate if the first two index values, the third and fourth index values, and the third and fifth index values of the variable are equivalent; and return the result. 
+    // Within if, create an else if statement that will evaluate if the first two index values, the first and third index values, and the fourth and fifth index values of the variable are equivalent; and return the result.
+    // Create an else statement that returns a false boolean
+// Run the function with the provided variables.
+
+let fullHouse = (hand) => {
+    let sortedHand = hand.sort((elementA, elementB) => elementA - elementB);
+    if(sortedHand[0] == sortedHand[1] && sortedHand[2] == sortedHand[3] && sortedHand[2] == sortedHand[4]) {
+        return 1 == 1
+    } else if(sortedHand[0] == sortedHand[1] && sortedHand[0] == sortedHand[2] && sortedHand[3] == sortedHand [4]) {
+        return 1 == 1
+    } else {
+        return 1 == 2
+    }
+}
+
+console.log(fullHouse(hand1))
+
+console.log(fullHouse(hand2))
+
+console.log(fullHouse(hand3))
```

Comment:
This is excellent.  You covered the other possible hand array that lists the pair first.  Really good thought process and excellent code.  Great job.  You can just return true or false, but I do like your line of thinking.  :)
Beautiful pseudocode!